src/api/python/genkinds.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env python
+"""
+This script reads CVC4/src/api/cvc4cppkind.h and generates
+.pxd and .pxi files which declare all the CVC4 kinds and
+implement a Python wrapper for kinds, respectively. The
+default names are kinds.pxd / kinds.pxi, but the name is
+configurable from the command line with --kinds-file-prefix.
+
+The script is aware of the '#if 0' pattern and will ignore
+kinds declared between '#if 0' and '#endif'.
```

Comment:
Can it also handle nested `#if 0` (or rather, `#if`* ) blocks? (It doesn't look like it, correct me if I'm wrong.)

Reply:
It cannot, but if that's important I could try to add that.

Reply:
So it actually does handle nested `#if 0` blocks, but no other macros at this point.

Reply:
As discussed offline with @makaimann , that's absolutely sufficient for us here.

Reply:
Can you maybe add a comment that it supports nested `#if 0` blocks?

Reply:
Definitely! Done with **0fa83b5910feb94a0587b4070ab6d44263d87d0c**